scientific_library/tvb/basic/config/settings.py

```diff
@@ -77,25 +79,33 @@ def __init__(self, manager, bin_folder):
         self.CODE_CHECKED_TO_VERSION = manager.get_attribute(stored.KEY_LAST_CHECKED_CODE_VERSION, -1, int)
 
     @property
-    def SVN_VERSION(self):
+    def GIT_VERSION(self):
         try:
             import tvb.basic.config
             config_folder = os.path.dirname(os.path.abspath(tvb.basic.config.__file__))
             with open(os.path.join(config_folder, 'tvb.version'), 'r') as version_file:
-                return self.parse_svn_version(version_file.read())
+                return self.parse_tvb_version(version_file.read())
         except Exception:
             pass
 
         return 42
 
     @staticmethod
-    def parse_svn_version(version_string):
+    def parse_tvb_version(version_string):
         if ':' in version_string:
             version_string = version_string.split(':')[1]
 
         number = ''.join([ch for ch in version_string if ch.isdigit()])
         return int(number)
 
+    @staticmethod
+    def fetch_current_revision(branch):
+        url = f'https://api.github.com/repos/the-virtual-brain/tvb-root/commits'
+        params = {'per_page': 1, 'sha': branch}
+        resp = requests.get(url, params)
+        last_link = resp.links.get('last')
+        return int(last_link['url'].split('&page=')[1])
```

Comment:
Here maybe we could add that static number that I mentioned (10.000) with the switch from SVN numbers to GIT count.

How would this API work in different branches ? Can we ensure a unique and incremental number is returned at two succesive calls?